### PR TITLE
Issue409 ashrae

### LINKED ISSUE #409: Add ASHRAE 140 test rooms and clean up package
This is to 
- [x] add test rooms of ASHRAE 140 to verification package 

and to clean that package by:

- [x] Fix typos in documentation
- [x] Rename VDI to `verification_VDI_6007_...`

### PR DESCRIPTION
**Pull request by mlauster**
This closes #409

Files for all four different validation cases

### CODE REVIEW

File: teaser/examples/verification/verification_ASHRAE_140_600.py [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+
+    prj.export_parameters_txt()
+    prj.export_annex()
+
+
+def from_scratch(number_of_elements, save=False,
+                 path=utilities.get_default_path()):
+    """
+    This function creates the test room from scratch.
+
```
Comment (PRemmen) [inline teaser/examples/verification/verification_ASHRAE_140_600.py]: Something like:

```python 
def from_scratch(
    number_of_elements, 
    save=False,
    path=utilities.get_default_path()):
    """This function creates the test room from scratch.
``` 
Would be better readable from my point of view

File: teaser/examples/verification/verification_ASHRAE_140_600.py [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+    win_2_material = Material(win_2_layer)
+    win_2_material.name = "GlasWindow"
+    win_2_material.thermal_conduc = 0.15
+    win_2_material.transmittance = 0.907
+    win_2_material.ir_emissivity = 0.9
+
+    #  This is a dummy ground floor to export three and four elements models.
+    #  Please set values for floor plate in three element and four element
+    #  models to default.
+
```
Comment (PRemmen) [inline teaser/examples/verification/verification_ASHRAE_140_600.py]: If #444 is merged we can probably delete the following few lines?

File: teaser/examples/verification/verification_ASHRAE_140_600.py [hunk 3]
```diff
@@ -0,0 +1,410 @@
...
+    return prj
+
+
+def load_file():
+
+    prj = Project(load_data=True)
+
+    prj.load_project(utilities.get_full_path(
+        "examples/examplefiles/ASHRAE140_600.teaserXML"))
```
Comment (PRemmen) [inline teaser/examples/verification/verification_ASHRAE_140_600.py]: it's better to use os.path.join()

File: teaser/examples/verification/verification_ASHRAE_140_620.py [hunk 4]
```diff
@@ -0,0 +1,410 @@
...
+    prj = Project(load_data=True)
+
+    prj.load_project(utilities.get_full_path(
+        "examples/examplefiles/ASHRAE140_600.teaserXML"))
+
+    return prj
+
+if __name__ == '__main__':
+    main()
+    print("ASHRAE 600: That's it!")
```
Comment (PRemmen) [inline teaser/examples/verification/verification_ASHRAE_140_620.py]: 620

File: teaser/examples/verification/verification_ASHRAE_140_920.py [hunk 5]
```diff
@@ -0,0 +1,411 @@
+# Created March 2017
+# TEASER Development Team
+
+"""
+This script contains of three functions. The first one loads the heavy-weight
+ASHRAE 140 test room 900 from a *.teaserXML file. The second one creates
```
Comment (PRemmen) [inline teaser/examples/verification/verification_ASHRAE_140_920.py]: 920

File: teaser/examples/verification/verification_ASHRAE_140_620.py [hunk 6]
```diff
@@ -0,0 +1,410 @@
...
+
+    return prj
+
+
+def load_file():
+
+    prj = Project(load_data=True)
+
+    prj.load_project(utilities.get_full_path(
+        "examples/examplefiles/ASHRAE140_600.teaserXML"))
```
Comment (PRemmen) [inline teaser/examples/verification/verification_ASHRAE_140_620.py]: no such file for test case 620 in repo

File: teaser/examples/verification/verification_ASHRAE_140_900.py [hunk 7]
```diff
@@ -0,0 +1,411 @@
...
+
+    return prj
+
+
+def load_file():
+
+    prj = Project(load_data=True)
+
+    prj.load_project(utilities.get_full_path(
+        "examples/examplefiles/ASHRAE140_900.teaserXML"))
```
Comment (PRemmen) [inline teaser/examples/verification/verification_ASHRAE_140_900.py]: no such file for test case 900in repo

File: teaser/examples/verification/verification_ASHRAE_140_920.py [hunk 8]
```diff
@@ -0,0 +1,411 @@
...
+
+    return prj
+
+
+def load_file():
+
+    prj = Project(load_data=True)
+
+    prj.load_project(utilities.get_full_path(
+        "examples/examplefiles/ASHRAE140_900.teaserXML"))
```
Comment (PRemmen) [inline teaser/examples/verification/verification_ASHRAE_140_920.py]: no such file for test case 920 in repo

File: teaser/examples/verification/verification_ASHRAE_140_600.py [hunk 9]
```diff
@@ -0,0 +1,410 @@
...
+def main():
+    number_of_elements = 4
+
+    prj = from_scratch(number_of_elements=number_of_elements, save=False)
+    # prj = load_file()
+
+    prj.buildings[0].calc_building_parameter(
+        number_of_elements=number_of_elements,
+        merge_windows=False,
+        used_library='Annex60')
```
Comment (mlauster) [inline teaser/examples/verification/verification_ASHRAE_140_600.py]: Done

File: teaser/examples/verification/verification_ASHRAE_140_600.py [hunk 10]
```diff
@@ -0,0 +1,410 @@
...
+    prj.used_library_calc = 'Annex60'
+    prj.number_of_elements_calc = number_of_elements
+    prj.merge_windows_calc = False
+    prj.weather_file_path = utilities.get_full_path(
+        os.path.join(
+            "data",
+            "input",
+            "inputdata",
+            "weatherdata",
+            "DEU_BW_Mannheim_107290_TRY2010_12_Jahr_BBSR.mos"))
```
Comment (mlauster) [inline teaser/examples/verification/verification_ASHRAE_140_600.py]: Added a weather file for ASHRAE 140

File: teaser/examples/verification/verification_ASHRAE_140_600.py [hunk 11]
```diff
@@ -0,0 +1,410 @@
...
+
+    prj.export_parameters_txt()
+    prj.export_annex()
```
Comment (mlauster) [inline teaser/examples/verification/verification_ASHRAE_140_600.py]: Done

File: teaser/examples/verification/verification_ASHRAE_140_600.py (hunk 1 above)
Comment (mlauster) [inline teaser/examples/verification/verification_ASHRAE_140_600.py]: Done

File: teaser/examples/verification/verification_ASHRAE_140_600.py (hunk 3 above)
Comment (mlauster) [inline teaser/examples/verification/verification_ASHRAE_140_600.py]: Done

File: teaser/examples/verification/verification_ASHRAE_140_620.py (hunk 6 above)
Comment (mlauster) [inline teaser/examples/verification/verification_ASHRAE_140_620.py]: Done

File: teaser/examples/verification/verification_ASHRAE_140_620.py (hunk 4 above)
Comment (mlauster) [inline teaser/examples/verification/verification_ASHRAE_140_620.py]: Done

File: teaser/examples/verification/verification_ASHRAE_140_920.py (hunk 5 above)
Comment (mlauster) [inline teaser/examples/verification/verification_ASHRAE_140_920.py]: Done

File: teaser/examples/verification/verification_ASHRAE_140_920.py (hunk 8 above)
Comment (mlauster) [inline teaser/examples/verification/verification_ASHRAE_140_920.py]: Done

File: teaser/examples/verification/verification_ASHRAE_140_900.py (hunk 7 above)
Comment (mlauster) [inline teaser/examples/verification/verification_ASHRAE_140_900.py]: Done

**Comment by mlauster** [comment]
@PRemmen, I fixed all points you mentioned. I encountered one more thing: i save all rooms as teaserXML files. If I try to load these files with the shipped function, I get the error that the year of construction is not defined. Is it really necessary to always define a year of construction?

**Comment by mlauster** [comment]
As far as I can judge, all new files are pep8 conform, how can I get more information why the tests fail?

**Comment by MichaMans** [comment]
@mlauster Sure! You can have a look in detail by clicking on the details button. Then you can see the builds and there you can look into the builds and into the failures.

**Comment by mlauster** [comment]
Yep, got it, thanks!

**Comment by PRemmen** [comment]
As mentioned above I would very much appreciate minimal unit tests that not necessarily test the results itself but check wether the verifications are running or not.

**Comment by mlauster** [comment]
@PRemmen, I added unit tests

**Comment by PRemmen** [comment]
@mlauster superb! ready to merge :)

**Pull request merged**